src/main/java/com/sysdig/jenkins/plugins/sysdig/containerrunner/DockerClientRunner.java

```diff
@@ -5,24 +5,58 @@
 import com.github.dockerjava.api.command.CreateContainerResponse;
 import com.github.dockerjava.api.model.Bind;
 import com.github.dockerjava.api.model.HostConfig;
+import com.github.dockerjava.core.DefaultDockerClientConfig;
 import com.github.dockerjava.core.DockerClientBuilder;
-import com.github.dockerjava.netty.NettyDockerCmdExecFactory;
+import com.github.dockerjava.core.DockerClientConfig;
+import com.github.dockerjava.httpclient5.ApacheDockerHttpClient;
+import com.github.dockerjava.transport.DockerHttpClient;
 import com.sysdig.jenkins.plugins.sysdig.log.SysdigLogger;
+import hudson.EnvVars;
 
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Properties;
 
 public class DockerClientRunner implements ContainerRunner {
 
   private final DockerClient dockerClient;
   private final SysdigLogger logger;
 
-  public DockerClientRunner(SysdigLogger logger) {
+  public DockerClientRunner(SysdigLogger logger, EnvVars currentEnv) {
+
+
+    DefaultDockerClientConfig.Builder configBuilder = DefaultDockerClientConfig.createDefaultConfigBuilder();
+    if (currentEnv.get("DOCKER_HOST") != null) {
```

Comment:
Unfortunately in previous commit this was done by just passing a set of Properties from the env vars, but the client builder will throw an exception when DOCKER_HOST is not defined. So we need to set the values individually instead of just providing the set of envvars and use the .withProperties() method.